### ISSUE TITLE
[Feature] Docker Package on GitHub

### ISSUE THREAD

**Issue by maxexcloo** [alucryd/oxyromon#114]
Wondering if it would be possible to use a GitHub action to build the Docker container in an automated fashion (similar to the image I attempted [here](https://github.com/maxexcloo/docker-oxyromon/blob/main/.github/workflows/docker.yml) before seeing your Dockerfile 😢).

Awesome project BTW, love how it seems to handle every kind of ROM and disk based system, game changer! <3

**Comment by maxexcloo** [comment]
(A nice example of how much more efficient it is in Arch Linux when comparing to my Ubuntu effort: https://github.com/maxexcloo/docker-oxyromon/blob/fe2c695710cdb84e7795f7e5d481c538d9f746c5/Dockerfile)

**Comment by alucryd** [comment]
Thanks for the kind words and donation :) Sure thing, I'll see what I can do.

I'm in the middle of a somewhat big refactoring to streamline future developments and avoid duplicated code, but I'll address your issues afterwards. Hopefully I'll be done by the end of the month.

**Comment by maxexcloo** [comment]
> Thanks for the kind words and donation :) Sure thing, I'll see what I can do.
> 
> I'm in the middle of a somewhat big refactoring to streamline future developments and avoid duplicated code, but I'll address your issues afterwards. Hopefully I'll be done by the end of the month.

More than happy to raise a PR with the code, if that helps? Will use your Dockerfile (had some issues building but I can likely fix tha!)

**Comment by alucryd** [comment]
Sure, PRs are welcome :)

**Comment by alucryd** [comment]
Well I managed to rewrite most of what I had in mind, so I went ahead and added the github action. You'll find a new `develop` tag on docker hub: https://hub.docker.com/repository/docker/alucryd/oxyromon

It's based on the archlinux dockerfile, I work for them so it's always going to be the most complete experience, alpine is lacking quite a few things in comparison. I'll keep publishing alpine images for stable releases, but I don't think it's necessary for the develop branch.
